mfkdf2/src/derive/factors/ooba.rs

```diff
@@ -80,10 +77,10 @@ pub fn ooba(code: String) -> MFKDF2Result<MFKDF2Factor> {
     factor_type: FactorType::OOBA(Ooba {
       target: vec![],
       length: 0,
-      jwk:    "".to_string(),
-      params: "".to_string(),
+      code:   code.to_uppercase(),
```

Comment:
added code to ooba factor struct. previously was adding it directly in params, which worked, but was not same as reference.